anvil-rstudio-base/README.md

```diff
@@ -1,19 +1,76 @@
+
+## Table of Contents
+- [RStudio Docker Contents](#rstudio-docker-contents)
+  * [Accessing the AnVIL RStudio Image](#accessing-the-anvil-rstudio-image)
+  * [Base Image](#base-image)
+  * [Applications](#applications)
+  * [Libraries](#libraries)
+  * [Customizations](#customizations)
+- [Using R Studio in Terra](#using-r-studio-in-terra)
+- [Saving RStudio Data](#saving-rstudio-data)
+- [Versioning](#versioning)
+
+## RStudio Docker Contents
+
+### Accessing the AnVIL RStudio Image
+
+The AnVIL Rstudio Image can be accessed using the following URL: us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.1. 
+
+### Base Image
+
+The base image for the AnVIL RStudio Docker is the [rocker/tidyverse](https://hub.docker.com/r/rocker/tidyverse/) image provided by the [Rocker Project](https://www.rocker-project.org/). This image additionally contains tidyverse and devtools.
+
+
+### Applications
+
+The AnVIL R Docker contains the [RStudio Server](https://www.rstudio.com/products/rstudio-server/) application which supports a web browser version of RStudio. 
+
+### Libraries
+
+The following packages are pre-loaded in the AnVIL RStudio image:
+
+* [tidyverse](https://www.tidyverse.org/packages/): a suite of packages designed for data sciences 
+* [google-cloud-sdk](https://cloud.google.com/sdk/): tools for the google cloud platform
+* [googleCloudStorageR](http://code.markedmondson.me/googleCloudStorageR/): an R library for interacting with google cloud storage
+* [bigrquery](https://github.com/r-dbi/bigrquery): a package to allow interaction with Google's BigQuery
+* [DataBiosphere/Ronaldo](https://github.com/DataBiosphere/Ronaldo): a package of functions commonly used for R Notebooks created from Leonardo
+
+### Customizations
+
+The AnVIL RStudio image is customized to disable the RStudio login screen. Future customizations will include:
+
+* RStudio hooks to refresh credentials 
+* An RStudio pause after inactivity
+
+## Using RStudio in Terra
+
+Detailed instructions on accessing the AnVIL RStudio image can be found in the "[Working with project-specific images](https://broadinstitute.zendesk.com/knowledge/articles/360037269472)" documentation in Terra support.
+
+## Saving RStudio Data
+
+Code and data generated in Terra RStudio will not be saved. Below are three suggestions for saving your work.
+
+### 1. Copying RStudio work to a workspace google bucket
+
+Use the [gsutil](https://cloud.google.com/storage/docs/gsutil) tool to copy files to your workspace google bucket. The "[Working with project-specific images](https://broadinstitute.zendesk.com/knowledge/articles/360037269472)" documentation in Terra support demonstrates how to identify the url for a workspace google bucket. After identifying the workspace google bucket url, you can copy files to a google bucket using the bash command: 
+
+    !gsutil cp ./* gs://GoogleBucketURL
```

Comment:
@rtitle is this correct? I thought `!` only works in jupyter notebooks.

I would also change `GoogleBucketURL` to `<WORKSPACE_BUCKET>` just so it's move obvious this is a variable here

Reply:
Changed to <WORKSPACE_BUCKET>

Reply:
Yeah, `!` is only for the Jupyter python kernel. 

But RStudio has a terminal, so this can just say to run the bash commands in the terminal (with no `!`).

Reply:
Changed the ! and mentioned using terminal